src/components/App/index.tsx

```diff
@@ -18,12 +18,6 @@ const offlinerPromise = import(
   '../../lib/offliner',
 );
 
-export interface SourceImage {
-  file: File | Fileish;
-  data: ImageData;
-  vectorImage?: HTMLImageElement;
-}
-
```

Comment:
I have no idea what this was doing here. Probably left over from when we split the compressor out into its own module.